src/session_client_tls.c

```diff
@@ -387,6 +388,9 @@ nc_client_tls_update_opts(struct nc_client_tls_opts *opts, const char *peername)
     char *key;
     X509_LOOKUP *lookup;
     X509_VERIFY_PARAM *vpm = NULL;
+    EVP_PKEY *pkey = NULL;
+    ENGINE *pkcs11 = NULL;
+    const char* pin = getenv("DEFAULT_USER_PIN");
```

Comment:
Is this a standard variable name used by other projects, and does it conform to how the OpenSSL API is "supposed to be used"? Where is the documentation?